azure-pipelines.yml

```diff
@@ -1,81 +1,129 @@
 variables:
   BUILD_OUTPUT_DIR: app/build/outputs/apk
+  TESTS_XML_DIR: app/build/outputs/androidTest-results
+  IMG_NAME: system-images;android-23;google_apis;x86
+  DEV_TYPE: "Nexus 4"
+  DEV_NAME: Nexus4Emu
+  DEV_VIDEO_DIR: /mnt/sdcard/DCIM
```

Comment:
Please use `DEVICE` prefix for these variables, otherwise it could be confused with 'developer'

Reply:
Done.